policyengine_us/variables/gov/states/ar/tax/income/credits/personal/ar_personal_credits.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_us.model_api import *
+
+
+class ar_personal_credits(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arkansas personal credits"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2021_AR1000F_FullYearResidentIndividualIncomeTaxReturn.pdf"
+        "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_FullYearResidentIndividualIncomeTaxReturn.pdf#page=1"
+        "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=12"
+    )
+    defined_for = StateCode.AR
+
+    # adds = [
+    #     "ar_aged_credit",
+    #     "ar_blind_credit",
+    #     "ar_deaf_credit",
+    #     "ar_dependent_credit",
+    #     "ar_status_credit",
+    # ]
+    adds = ["ar_non_refundable_credits", "ar_refundable_credits"]
```

Comment:
we can not add together all refundable and non_refundable credits - this includes more than just personal credits

Reply:
we need a **list parameter** with #     "ar_aged_credit",
    #     "ar_blind_credit",
    #     "ar_deaf_credit",
    #     "ar_dependent_credit",
    #     "ar_status_credit",

which then is added here

Reply:
additionally, we need to add the variable ar_personal_credits to a lost of refundable **or** non-refundable credits which will include other credits as well

Reply:
please review the documentation and the code of other developers